frontend/src/services/axios/axios.ts

```diff
@@ -0,0 +1,37 @@
+import axios from "axios"
```

Comment:
Using axios is okay. better would be to use a client generated by the backend. There is also a issue for it [#12](https://github.com/world-wide-weights/wwweights/issues/12)

Shall we use axios now and update later?